web/components/Users/Search.tsx

```diff
@@ -106,7 +117,8 @@ export default function Search({
                         label: val,
                       })
                     )}
-                    onChange={(newVals) =>
+                    onChange={async (newVals) => {
+                      await logFilterClickEvent('Foster Type');
```

Comment:
Nit: Consider removing the `await` here and just call the logging function as normal since the code after it doesn't require it to proceed